presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitManager.java

```diff
@@ -135,7 +136,8 @@ public HiveSplitManager(
                 hiveClientConfig.getMaxInitialSplitSize(),
                 hiveClientConfig.getMaxInitialSplits(),
                 hiveClientConfig.isForceLocalScheduling(),
-                false);
```

Comment:
Swap the order of these everywhere, so that `recursiveDfsWalkerEnabled` stays last, since currently it is not configurable (it is basically a constant).